src/ios/CDVSearchAds.swift

```diff
@@ -28,7 +27,7 @@ import AdServices
     }
 
   func fetchAttributionData(command:CDVInvokedUrlCommand) {
-      if #available(iOS 15, *) {
+      if #available(iOS 14.3, *) {
```

Comment:
Why the downgrade?

Reply:
Not a downgrade: casting the net slightly more widely now that the deprecated framework is no long available. `AAAttribution.attributionToken()` was introduced in iOS 14.3

Reply:
However, I now notice that I deliberately changed this because of issues with iOS 14.5: https://github.com/Roosterbank/cordova-plugin-searchads/commit/831c709b8ce08aebdd82e9c05f9b321b12f7e3af

Reply:
Phew, thank goodness you asked that question @guideveloper